inc/config.class.php

```diff
@@ -3805,4 +3822,19 @@ static function getIcon() {
       self::setConfigurationValues('core', [$type . '_uuid' => $uuid]);
       return $uuid;
    }
+
+   /**
+    * @param $input
+    *
+    * @return string
+    */
+   static function _serialize($input) {
```

Comment:
I do not know what you are trying to achieve, but there already are methods that handle json, and this is not relevant in this class anyways.

Reply:
corrected